source/blog/2022-11-17-announcing-hanami-200rc.html.markdown

```diff
@@ -0,0 +1,357 @@
+---
+title: "Hanami 2.0: Awesome Short Summary Here"
+date: 2022-11-17 08:24:53 UTC
+tags: announcements
+author: Tim Riley
+image: true
+excerpt: >
+  Introducing a distinct and compelling new vision for Ruby apps of all shapes and sizes.
+---
+
+After more than three years of work, Hanami 2.0 is here! This release marks a new era for Hanami, and presents a revolutionary new vision for Ruby applications!
+
+## What is Hanami, and why does 2.0 matter?
+
+Since our beginning we’ve called Hanami a _modern web framework for Ruby_. With today’s release, it becomes so much more. Hanami 2.0 is **the framework for Ruby apps of all shapes and sizes.**
+
+With Hanami 2.0 you can still have your “hello world” app running within minutes, but you’ll be doing so using a framework that provides everything you need for your app to grow without sacrificing maintainability. With its innovative component management, Deps mixin and slices, Hanami is the first Ruby framework to acknowledge the **critical need to organise your business logic**, and ship with all the tools necessary to do so.
+
+The majority of the code you’ll write in an Hanami app will be glorious plain old Ruby. **This is a framework that works in service of your code, rather than the other way around.** Hanami’s own facilities, such as the router and actions, are designed to occupy their own distinct layers that decorate and call into your code.
```

Comment:
```suggestion
The majority of the code you’ll write in a Hanami app will be glorious plain old Ruby. **This is a framework that works in service of your code, rather than the other way around.** Hanami’s own facilities, such as the router and actions, are designed to occupy their own distinct layers that decorate and call into your code.
```